sale_packaging_default/models/sale_order_line.py

```diff
@@ -22,6 +22,9 @@ def onchange(self, values, field_name, field_onchange):
     @api.depends("product_id", "product_uom_qty", "product_uom")
     def _compute_product_packaging_id(self):
         """Set a default packaging for sales if possible."""
+        if not self.env.context.get("apply_default_packaging", True):
```

Comment:
Depending on context for stored fields is usually bad. Maybe you can explain a little bit your use case ?

Reply:
But Is see @yajo has already asked.